tests/glm/test_glm.py

```diff
@@ -2260,6 +2268,50 @@ def test_store_covariance_matrix(
     )
 
 
+@pytest.mark.parametrize(
+    "formula", ["y ~ col_1 + col_2", "col_1 + col_2"], ids=["two-sided", "one-sided"]
+)
+def test_store_covariance_matrix_formula(regression_data, formula):
+    X, y = regression_data
+    df = pd.DataFrame(X, columns=[f"col_{i}" for i in range(X.shape[1])])
+
+    if "~" in formula:
```

Comment:
always the case in this test, right?

Reply:
No, there are two test cases and the formula in the second one is simply `"col_1 + col_2"`. Formatting is a bit wonky though and it's hard to parse, but I blame black for that 😄